include/apps/tools/OswAppToolsPage2App.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <osw_hal.h>
+#include <osw_ui.h>
+#include "osw_app.h"
+
+class OswAppToolsPage2App : public OswApp {
+  public:
+    OswAppToolsPage2App(void) {
+        ui = OswUI::getInstance();
+    };
+    virtual void setup() override;
+    virtual void loop() override;
+    virtual void stop() override;
+    ~OswAppToolsPage2App() {};
+
+  private:
+    OswUI* ui;
+};
```

Comment:
Same as the other class - additionally, have you heard of `template` classes?

Reply:
yes, there is a plan to replace the template code with functional code, for say - various tools.. I will keep working on things and can re-submit something more appropiate. I did move some other apps around and it seemed to be more convenient when using the watch.